src/main/java/rife/continuations/instrument/TypesMethodVisitor.java

```diff
@@ -843,7 +743,7 @@ private void processInstructions(TypesNode node) {
         TypesContext exception_context = null;
         for (var instruction : node.getInstructions()) {
             switch (instruction.getOpcode()) {
-                case TypesOpcode.SET: {
+                case TypesOpcode.SET -> {
```

Comment:
Same here, I'd rather not make this enum change and have the opcodes formatted like the other enum above

Reply:
Reverted, although on that one I fail to see where it makes any difference in readability.

Reply:
It just brings it in line again with the formatting of the switch statement in the method above